scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
Works for me. If someone is interested in writing up some bash and parallel instead of using gox, I think they will be equivalent. I don't necessarily see it as a huge advantage over running a fork of gox, but it'll work just as well.